harness/temporalHelpers.js

```diff
@@ -101,7 +103,7 @@ var TemporalHelpers = {
    * the corresponding field in another Temporal.Duration.
    */
   assertDurationsEqual(actual, expected, description = "") {
-    assert(expected instanceof Temporal.Duration, `${description} expected value should be a Temporal.Duration`);
+    assert(expected instanceof Temporal.Duration, `${description}: expected value should be a Temporal.Duration`);
```

Comment:
Why not do the prefix dance here? Also e.g. `assertInstantsEqual`

Reply:
Thanks for catching that.  I think I just overlooked them.